source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
+This tutorial assumes you have successfully installed :ref:`Clear Linux on bare metal<bare-metal-install>` and that you have :ref:`set up a LAMP web server <web-server-install>`.
 
-Additionally, all the steps on this tutorial were tested using a NUC6i5SYH
-Intel® NUC. Visit the `NUC6i5SYH product page`_ for detailed information.
+Before you install new packages, update the |CL| OS with the following command:
 
-Before installing any new packages, update the |CL| OS with the
-console command:
-
-.. code-block:: console
+.. code-block:: bash
 
    sudo swupd update
 
 
-Creating a WordPress server
+Create a WordPress server
 ===========================
 
-This tutorial walks you through the process of installing the WordPress
-components on your |CL| LAMP server. At the end of the
-:ref:`web server tutorial <web-server-install>`, we created the initial
-WordPress MySQL database `WordPress` using phpMyAdmin\* and MariaDB\*.
-With the LAMP server up and running, we can add the WordPress components
-needed to host a WordPress website on your host system.
-
-Throughout this tutorial we will reference your website name as
-<your_website>. There are numerous articles available on-line regarding how
-to name your website and acquire the necessary certificates. Those tasks are
-beyond the scope of this tutorial.
-
-There are also several actions you can take to harden your website from
-attacks. The security of your website and the data it contains are complex
-and ever-evolving tasks. They should be at the top of your to do list if you
-plan to expose your website to the outside world. This tutorial does not
-address security measures that you can take to harden your site but we
-strongly encourage you to look into it.
-
-Installing WordPress
---------------------
+WordPress can be installed in a variety of ways. These instructions are written for users who have installed phpMyAdmin.
 
-The folks at WordPress have created a `step-by-step guide`_ to install
-WordPress quickly and easily. We are following this procedure very closely.
+This tutorial follows these steps:
 
-We are setting up a WordPress blog which can be accessed at:
+1. Download WordPress and add components to the necessary directory. 
+2. Set up WordPress by entering information in the WordPress admin console.
+3. Run the installer to fully install WordPress on your system and create a username and password for your newly created blog. 
+
+Note that throughout this tutorial, we reference your website name as <your_website>. 
 
-http://<your_website>/blog
 
-To accomplish this setup, we must put the WordPress components in the
-:file:`/var/www/html/blog` directory.
+Install WordPress
+--------------------
+
+For this tutorial, we will create a WordPress blog that can be accessed at:
+
+http://<your_website>/blog
 
-Let us get started:
+To accomplish this setup, you must add WordPress components to the :file:`/var/www/html/blog` directory.
 
+Follow these steps:
 
```

Comment:
Add header: 

Download WordPress and manage directory
------------------------------------------------

Reply:
Done